fieldservice_agreement/views/agreement_view.xml

```diff
@@ -32,6 +32,14 @@
                 </button>
             </div>
             <notebook position="inside">
+                <page name="serviceprofiles" string="Service Profiles">
+                    <field name="serviceprofile_ids" 
+                           context="{'default_agreement_id': active_id}">
+                        <tree>
+                            <field name="name"/>
+                        </tree>
+                    </field>
+                </page>
```

Comment:
@osi-scampbell Should be moved to `agreement_serviceprofile` if not already there.